blog/_posts/TeamDelSquared.md

```diff
@@ -0,0 +1,40 @@
+### Team DelSquared: Our Summer of Code, 2018
+
+ **And...it’s a wrap!**
+ 
+September is approaching to an end and we still can’t believe how fast the last three months have gone by. This is Arushi and Sharmistha, from India, and we worked this summer on the Tessel project. The experience has been simply wonderful. Sitting for hours trying to figure out how to go about our first pull request, the weekly and monthly meetings with our team, closing the first issue we worked on, receiving the package with Tessel modules (and stickers of course!!!) - these are a few of the many instances that have made our summer a truly memorable experience.
+Back when we first started working with Tessel, we were really really excited about being able to actually contribute to open source, because that meant collaboration with so many people around the world. The excitement hasn’t gone away yet, even as the end of the program approaches, and we intend to continue working with Tessel and mentor the newcomers. During this program, we actually learnt quite a lot about how such developments boards work. Before that, everything for us was like a black box. Feed code, and run the program. Not anymore.
+After having delved neck deep into online resources, tutorials and repositories on GitHub, we have gained a lot of related knowledge on hardware, communication protocols, and even hardware. Here are some highlights of the work we did:
+Worked on an ESP32 Hello World tutorial for Windows 10 and submitted a blogpost for the same. (currently under review)
+Had a mini Tessel hackathon in which we tried to build a SoundBot of sorts. We controlled LED bulbs with live music and made them fluctuate in sync with the music beats. We used a Tessel board and an ambient module for the same. The blogpost is currently under review and would be out soon!
+Figured out a way to build diagrams of the Tessel components using Fritzing and submit them in the required format. We really had to bang our hands for this one. It was proving to be pretty annoying :p 
+Worked on a few documentation issues, some of which we were able to close while others are under review or need to be updated.
+Learnt a lot about the I2C protocol as we had to browse a lot of online resources to learn how devices talk to each other using these communication protocols. We are currently working on implementing I2C to create a demo for communication between an ESP32 and an accelerometer module.
+Worked with several APIs and softwares like Moddable, ImageMagick. We’re still exploring Moddable but it seems like a amazing tool for communicating to micro-controllers which are based on JavaScript.
+
+ **Looking Ahead** </br>
```

Comment:
Same comment as above. Break the long paragraph into smaller bullet points of paragraphs. Also, please check the preview before sending a revised commit. For example, tag '</br>' doesn't seem to be working here. You can figure that out if you check the output.